runtime/gc_base/GCExtensions.hpp

```diff
@@ -149,14 +149,17 @@ class MM_GCExtensions : public MM_GCExtensionsBase {
 	UDATA deadClassLoaderCacheSize;
 #endif /*defined(J9VM_GC_DYNAMIC_CLASS_UNLOADING) */
 
-
+	bool statupPhaseFinished; /**< Used to indicate if the JIT startup phase is finished */
 
 	MM_UnfinalizedObjectList* unfinalizedObjectLists; /**< The global linked list of unfinalized object lists. */
 	
 	UDATA objectListFragmentCount; /**< the size of Local Object Buffer(per gc thread), used by referenceObjectBuffer, UnfinalizedObjectBuffer and OwnableSynchronizerObjectBuffer */
 
 	MM_Wildcard* numaCommonThreadClassNamePatterns; /**< A linked list of thread class names which should be associated with the common context */
 
+#if defined(J9VM_GC_VLHGC)
+	UDATA tarokTargetMaxPauseTime; /**< An optional, user specified soft max pause time for PGC's in balanced GC*/
```

Comment:
is there a group of tarok specific parms already that you could move this to?

Reply:
There aren't any VLHGC/tarok params elsewhere in this file

Reply:
ok, move then to OMR/Base super class, where we have lots of them
while it's really semantically less correct, at least we are consistently wrong for all those params

Reply:
Moved it to `GCExtensionsBase` as per latest commit on OMR side https://github.com/eclipse/omr/pull/5825